docs/content/concepts/automation/schedules/troubleshooting.mdx

```diff
@@ -0,0 +1,140 @@
+
+</TabItem>
+<TabItem name="Dagster+">
+
+1. In the UI, click **Deployment** in the top navigation.
+2. In the **Code locations** tab, locate the code location that contains the schedule definition.
+3. Click **Redeploy**.
+
+</TabItem>
+</TabGroup>
+
+**If the code location can't be loaded** - for example, due to a syntax error - it will have a **Status** of **Failed**. Click the **View error** link in this column to view the error message.
+
+**If the code location loaded successfully** but the schedule isn't present in the **Schedules** tab, the schedule may not be included in the code location's `Definitions` object. Refer to [Step 1](#step-1-verify-the-schedule-is-included-in-the-definitions-object) for more information.
+
+---
+
+## Step 4: Verify dagster-daemon setup
+
+<Note>This section is applicable to Open Source (OSS) deployments.</Note>
+
+If the schedule interval is correctly configured but runs aren't being created, it's possible that the dagster-daemon process isn't working correctly. If you haven't set up a Dagster daemon yet, refer to the [Deployment guides](/deployment) for more info.
+
+### Verify the daemon is running
+
+1. In the UI, click **Deployment** in the top navigation.
+2. Click the **Daemons** tab.
+3. Locate the **Scheduler** row.
+
+The daemon process periodically sends out a hearbeat from the scheduler. If the scheduler daemon has a status of **Not running**, this indicates that there's an issue with your daemon deployment. If the daemon ran into an error that resulted in an exception, this error will often display in this tab.
+
+If there isn't a clear error on this page or if the daemon should be sending heartbeats but isn't, move on to the next step.
+
+### Check the daemon process logs
+
+Next, check the logs from the daemon process. The steps to do this will depend on your deployment - for example, if you're using Kubernetes, you'll need to get the logs from the pod that's running the daemon. You should be able to search those logs for the name of the schedule (or `SchedulerDaemon` to see all logs associated with the scheduler) to gain an understanding of what's going wrong.
+
+If the daemon output contains error indicating the schedule couldn't be found, verify that the daemon is using the same `workspace.yaml` file as the webserver. The daemon does not need to restart in order to pick up changes to the `workspace.yaml` file. Refer to the [Workspace files documentation](/concepts/code-locations/workspace-files) for more information.
+
+If the logs don't indicate the cause of the issue, move on to the next step.
+
+### Check for execution failures
+
+1. In the Dagster UI, click **Overview > Schedules tab**.
+2. Locate the schedule.
+3. In the **Last tick** column, check for a **Failure** badge. If the schedule failed to execute, you can click the badge to display the error and stack trace describing the failure.
```

Comment:
Should this be in the "daemon" section, or possibly moved to step 2?

Reply:
I wondered that too, tbh. This content pre-dates me for the most part, but this feels like something the user should do when they're already in that section. I'll add something about it there and mention the daemon as a possible cause if using OSS